app/UI/app_update.tsx

```diff
@@ -640,6 +650,47 @@ export function generateTags(content: string, getEventByID: func_GetEventByID, c
             }
         }
     }
+    if (reply_to_event_id) {
```

Comment:
Maybe we can simplify this to just `e` reply and `p` tagged pubkey. The current logic is hard to follow.